bot/exts/recruitment/talentpool/_review.py

```diff
@@ -0,0 +1,316 @@
+            last_infr, last_count = infr_stats[-1]
+            infractions += f", and {last_count} {self._format_infr_name(last_infr, last_count)}"
+
+        infractions = f"**{infractions}**"
+
+        # Show when the last one was issued.
+        if len(infraction_list) == 1:
+            infractions += ", issued "
+        else:
+            infractions += ", with the last infraction issued "
+
+        # Infractions were ordered by time since insertion descending.
+        infractions += get_time_delta(infraction_list[0]['inserted_at'])
+
+        return f"They have {infractions}."
+
+    @staticmethod
+    def _format_infr_name(infr_type: str, count: int) -> str:
+        """
+        Format the infraction type in a way readable in a sentence.
+
+        Underscores are replaced with spaces, as well as *attempting* to show the appropriate plural form if necessary.
+        This function by no means covers all rules of grammar.
+        """
+        formatted = infr_type.replace("_", " ")
+        if count > 1:
+            if infr_type.endswith(('ch', 'sh')):
+                formatted += "e"
+            formatted += "s"
+
+        return formatted
+
+    async def _previous_nominations_review(self, member: Member) -> Optional[str]:
+        """
+        Formats the review of the nominee's previous nominations.
+
+        The number of previous nominations and unnominations are shown, as well as the reason the last one ended.
+        """
+        log.trace(f"Fetching the nomination history data for {member.id}'s review")
+        history = await self.bot.api_client.get(
+            self._pool.api_endpoint,
+            params={
+                "user__id": str(member.id),
+                "active": "false",
+                "ordering": "-inserted_at"
+            }
+        )
+
+        if not history:
+            return
+
+        num_entries = sum(len(nomination["entries"]) for nomination in history)
+
+        nomination_times = f"{num_entries} times" if num_entries > 1 else "once"
+        rejection_times = f"{len(history)} times" if len(history) > 1 else "once"
+        end_time = time_since(isoparse(history[0]['ended_at']).replace(tzinfo=None), max_units=2)
+
+        review = (
+            f"They were nominated **{nomination_times}** before"
+            f", but their nomination was called off **{rejection_times}**."
+            f"\nThe last one ended {end_time} with the reason: {history[0]['end_reason']}"
+        )
+
+        return review
+
+    @staticmethod
+    def _random_ducky(guild: Guild) -> Union[Emoji, str]:
+        """Picks a random ducky emoji to be used to mark the vote as seen. If no duckies found returns 👀."""
+        duckies = [emoji for emoji in guild.emojis if emoji.name.startswith("ducky")]
+        if not duckies:
+            return "👀"
+        return random.choice(duckies)
+
+    @staticmethod
+    async def _bulk_send(channel: TextChannel, text: str) -> List[Message]:
```

Comment:
It could, but I'm not sure if it's a good idea to load the file with functions that are only used in a single file and could pass as just simple helper functions. We can move it when needed, like I did with get_time_delta in this PR

Reply:
Sounds good! :D